src/index.d.ts

```diff
@@ -1317,12 +1317,18 @@ declare namespace admin.machineLearning {
     /**
      * Wait for the model to be unlocked.
      *
-     * @param {number} maxTimeSeconds The maximum time in seconds to wait.
+     * @param {number} maxTimeMillis The maximum time in milliseconds to wait.
+     *     If not specified, a default maximum of 2 minutes will be used.
      *
      * @return {Promise<void>} A promise that resolves when the model is unlocked
      *   or the maximum wait time has passed.
      */
-    waitForUnlocked(maxTimeSeconds?: number): Promise<void>;
+    waitForUnlocked(maxTimeMillis?: number): Promise<void>;
+
+    /**
+     * Return the JSON format of the model.
```

Comment:
This feels slightly weird, unless "format" is a special noun in your context.

Would "Return the model in JSON format" or "as a JSON object" work?

Reply:
done